File: src/main/java/lesson12/Person.java
package lesson12;

@DbTable(name = "person")
class Person {

    @DbId
    @DbColumn
    Long id;
    @DbColumn
    String fio;
    @DbColumn
    String phone;
    @DbColumn
    Integer age;

    public Person() {
    }

    public Person(String fio, String phone, Integer age, Long tempData) {
        this.fio = fio;
        this.phone = phone;
        this.age = age;
        this.tempData = tempData;
    }

    @Override
    public String toString() {
        return "Person{" +
                "id=" + id +
                ", fio='" + fio + '\'' +
                ", phone='" + phone + '\'' +
                ", age=" + age +
                ", tempData=" + tempData +
                '}';
    }

    Long tempData;
}

File: src/main/kotlin/lesson12_kt/Person.kt
package lesson12_kt

@DbTable("person")
data class Person (
    @DbId
    @DbColumn
    @JvmField var id : Long = 0,
    @DbColumn
    @JvmField var fio: String = "",
    @DbColumn
    @JvmField var phone: String = "",
    @DbColumn
    @JvmField var age: Int = 0,
    @JvmField var cache: String = "",
)